packages/tame-metering/package.json

```diff
@@ -4,13 +4,14 @@
   "description": "tame-metering",
   "main": "src/index.js",
   "scripts": {
-    "test": "tape -r esm 'test/**/*.js'",
+    "test": "tap --no-coverage --reporter specy 'test/**/test*.js'",
```

Comment:
In ses-shim, the prevailing pattern is `test/**/*.test.js`. I assume this matches your test layout in `tame-metering`. I’ll have to fiddle with `specy` over `spec`.

Reply:
I prefer `test*.js`, but `tame-metering` is one of the few (only?) packages that has e.g. `circular.js` and `sanity.js` as test files, rather than as libraries that the real test files import from.

I'll file a cleanup ticket to rename those files, and unify all our `test` runner scripts to use the same glob.

For most of the packages that I switched to `tap`, I used the default reported. I didn't mean to leave this one with `specy` (I think I inherited that from JF in the SES-shim repo, but I'm not sure I have a preferred reporter yet, so I figured I should start with the default). I'll change this one to use the default too.

Reply:
Er, `bundle-source` has `circular.js`. `tame-metering` has only `test-XYZ.js` -style names. I'll change the glob.

Reply:
added https://github.com/Agoric/agoric-sdk/issues/1211